fleet/agent_service.go

```diff
@@ -14,6 +14,7 @@ import (
 	"github.com/mainflux/mainflux"
 	mfsdk "github.com/mainflux/mainflux/pkg/sdk/go"
 	"github.com/ns1labs/orb/fleet/backend"
+	"github.com/ns1labs/orb/fleet/backend/pktvisor"
```

Comment:
No this module can't reference pktvisor or pktvisor concepts. So, `ViewAgentBackendHandler`, `ViewAgentBackendInput` and `ViewAgentBackendTaps` should not be service methods here

Reply:
You're... it was my bad, but those guys shouldn't be here... it's just trash. All this guys was moved to the pktvisor backend.